common.py

```diff
@@ -69,10 +69,9 @@ def __init__(self, dictionary):
     def write(self):
         """This function writes the data to elastic search"""
         self.dictionary = json.dumps(self.dictionary)
-        requests.post('%s/%s-%s/metric_data/' %
-                      ('http://elasticsearch2.gridpp.rl.ac.uk:9200',
-                       'logstash-gridtools-metrics',
-                       date),
+        requests.post(("http://elasticsearch2.gridpp.rl.ac.uk:9200/"
```

Comment:
I think the extra brackets are unnecessary. So long as a string split over a number of lines is within some kind of bracket (i.e. the `post` argument) it should be treated as one. Would need to double check.

Reply:
+1